style.css

```diff
@@ -0,0 +1,634 @@
+body {
+	font-family: 'Gotham Pro';
+	font-size: 14px;
+	max-width: 1030px;
+	min-width: 320px;
+	margin: 0 auto;
+	padding: 32.46px 0;
+}
+
+a {
+	text-decoration: none;
+	color: #000000;
+	text-transform: uppercase;
+}
+
+a:hover {
+	opacity: .5;
+}
+
+/*<header class='header'>*/
+.header {
+	color: #000000;
+	font-size: 12px;
+	font-weight: bold;
+	display: flex;
+	justify-content: space-between;
+	margin: 0 0 29px 0;
+}
+
+.header__conteiner {
+	flex: 0 1 50%;
+	display: flex;
+	align-items: center;
+}
+
+.header__img {	
+	margin: 0 45px 0 0;
+}
+.header__logo {
+	height: 55px;
+	width: 55px;
+}
+
+.header__logo-mobile {display: none;}
```

Comment:
Codacy found an issue: [Expected single space before "}" of a single-line block (block-closing-brace-space-before)](https://app.codacy.com/gh/WebHeroSchool/ProjectMod2_EKh/pullRequest?prid=6247910)